### ISSUE TITLE
Pagination for proposal credit table

### ISSUE THREAD

**Issue by thi4go** [decred/politeiagui#615]
Right now, if we do many proposal credit transactions, our manage modal gets really big. We need to add a frontend-only pagination for the table, for better ux when the user has many recorded transactions.

**Comment by MariaPleshkova** [comment]
@thi4go are you talking about this one?
![image](https://user-images.githubusercontent.com/8841018/51253540-a2094b00-19af-11e9-8f40-d81c28a8fac7.png)

**Comment by thi4go** [comment]
Yep, that's the one @MariaPleshkova! I believe a good number to paginate is when it's down to ten transactions or so, to avoid a rendering of a giant modal.

**Comment by MariaPleshkova** [comment]
Created a design for it
**Preview:** https://xd.adobe.com/view/10b35c65-df17-4d0f-5e03-b946c5883912-6682/
**Specs:** https://xd.adobe.com/spec/bf76e2d6-dbc7-494a-536a-607950787724-c947/
![image](https://user-images.githubusercontent.com/8841018/51379533-aef97c00-1b20-11e9-9409-d3938ad808e9.png)
